.gitignore

```diff
@@ -1,2 +1,2 @@
 
-.DS_Store
+*.DS_Store
```

Comment:
👁️

Reply:
It was correct, you have to `git rm` already checked in files

Reply:
yep, I did. Don't think broke anything, but can change it back